js/actions/tests/newSource.js

```diff
@@ -1,25 +1,28 @@
 "use strict";
 
-const { createStore, actions, queries } = require("../../util/test-head");
+const { createStore } = require("../../util/test-head");
+const { newSource } = require("../../actions");
+const { getSourceByActor } = require("../../queries");
+
 const store = createStore();
 var expect = require("chai").expect;
```

Comment:
Honestly I can't really remember the real reason I switched a while ago. Maybe we should see what tofino is using; if we don't care too much it'd be nice to be on the same page for some stuff like this